site/pages/api/publish.api.ts

```diff
@@ -60,22 +62,30 @@ const publish = async (req: NextApiRequest, res: NextApiResponse) => {
                 res,
             );
 
-            redirectTo(res, `${REVIEW_DISRUPTION_PAGE_PATH}/${validatedBody.data.disruptionId}`);
+            redirectToWithQueryParams(
+                req,
+                res,
+                template ? ["template"] : [],
+                `${REVIEW_DISRUPTION_PAGE_PATH}/${validatedBody.data.disruptionId}`,
+            );
             return;
         }
 
         await insertPublishedDisruptionIntoDynamoAndUpdateDraft(
             draftDisruption,
             session.orgId,
-            canPublish(session) ? PublishStatus.published : PublishStatus.pendingApproval,
+            canPublish(session) || draftDisruption.template ? PublishStatus.published : PublishStatus.pendingApproval,
```

Comment:
I'm not sure if the logic around PublishStatus is working correctly, in my dynamo table I don't have the column publishStatus at all.

Reply:
done